### ISSUE TITLE
Automatic deployment of study 1 branch

### ISSUE THREAD

**Issue by zven** [sitcomlab/simport-learning-app#198]
This card combines the deployment setup created in https://github.com/sitcomlab/simport-learning-app/issues/134 and the extended feature flags and configuration implemented here https://github.com/sitcomlab/simport-learning-app/issues/187.

**Comment by felixerdy** [comment]
I'll wait until #202 and #203 is merged. Then, I will create a branch called `study-1` with the according feature flags and then we should be able to check whether the github action (which publishes the `study-1` branch to the play store) runs correctly

**Comment by zven** [comment]
@felixerdy https://github.com/sitcomlab/simport-learning-app/pull/202 and https://github.com/sitcomlab/simport-learning-app/pull/203 are merged. And https://github.com/sitcomlab/simport-learning-app/pull/204 should have fixed the remaining issue with the keystore. This card is basically done then, once the current develop state is merged into the studey-1 branch, right?

**Comment by zven** [comment]
Workflow run fails: https://github.com/sitcomlab/simport-learning-app/runs/6833031563?check_suite_focus=true

This seems to be related to the latest version of the library we are using for uploading the artifact to google play: https://github.com/r0adkll/upload-google-play/issues/85

**Comment by zven** [comment]
Workflow run fails again: `Changes cannot be sent for review automatically. Please set the query parameter changesNotSentForReview to true. Once committed, the changes in this edit can be sent for review from the Google Play Console UI.` https://github.com/sitcomlab/simport-learning-app/runs/6877517537?check_suite_focus=true#step:4:15

Apparently related to the fact that we have a rejected version in the closed test track: https://github.com/Triple-T/gradle-play-publisher/issues/956#issuecomment-847691895 As long as the version is rejected (and this issue is not resolved), you are apparently also not able to upload a new version for internal tests.

**Comment by felixerdy** [comment]
I just documented Googles comments to our rejected app in issue #208

**Comment by felixerdy** [comment]
After merging the latest changes of the diary export fix (#221) in the `study-1` branch the GitHub Action now successfully published the app as an internal release in the google play console: https://github.com/sitcomlab/simport-learning-app/runs/7687823561?check_suite_focus=true
